### ISSUE TITLE
Unknown painting motive

### ISSUE THREAD

**Issue by edwinyoo44** [pmmp/PocketMine-MP#5099]
### Issue description

- Expected result: What were you expecting to happen?
Place the painting and save it in the map
- Actual result: What actually happened?
After placing the painting, restarting the server cannot be saved in the map

### Steps to reproduce the issue
1. Place the painting
2. Restart the server
3. Jointhe game

### OS and versions

* PocketMine-MP: PocketMine-MP 4.4.1
* PHP:8.0.17
* Using JIT: no
* Server OS: Ubuntu 20.04
* Game version: Android

### Plugins
no

### Crashdump, backtrace or other files
```
[16:05:00.047] [Server thread/ERROR]: [World: Resources-12] [Loading chunk 324 358] Bad entity data at list position 3: Unknown painting motive
[16:05:00.047] [Server thread/CRITICAL]: pocketmine\data\SavedDataLoadingException: "Unknown painting motive" (EXCEPTION) in "pmsrc/src/entity/EntityFactory" at line 131
--- Stack trace ---
  #0 pmsrc/src/entity/EntityFactory(241): pocketmine\entity\EntityFactory->pocketmine\entity\{closure}(object pocketmine\world\World#112834, object pocketmine\nbt\tag\CompoundTag#138227)
  #1 pmsrc/src/world/World(2488): pocketmine\entity\EntityFactory->createFromData(object pocketmine\world\World#112834, object pocketmine\nbt\tag\CompoundTag#138227)
  #2 pmsrc/src/world/World(2443): pocketmine\world\World->initChunk(int 324, int 358, object pocketmine\world\format\io\ChunkData#138260)
  #3 pmsrc/src/world/World(665): pocketmine\world\World->loadChunk(int 324, int 358)
  #4 pmsrc/src/world/World(2830): pocketmine\world\World->registerChunkLoader(object anonymous@pmsrc/src/world/World#L2829#138129, int 324, int 358)
  #5 pmsrc/src/world/World(2855): pocketmine\world\World->checkChunkPopulationPreconditions(int 324, int 358)
  #6 pmsrc/src/Server(574): pocketmine\world\World->requestChunkPopulation(int 324, int 358, null)
  #7 pmsrc/src/network/mcpe/NetworkSession(221): pocketmine\Server->createPlayer(object pocketmine\network\mcpe\NetworkSession#137825, object pocketmine\player\XboxLivePlayerInfo#137927, true, object pocketmine\nbt\tag \CompoundTag#137864)
  #8 pmsrc/src/network/mcpe/NetworkSession(682): pocketmine\network\mcpe\NetworkSession->createPlayer()
  #9 pmsrc/src/network/mcpe/handler/ResourcePacksPacketHandler(132): pocketmine\network\mcpe\NetworkSession->pocketmine\network\mcpe\{closure}()
  #10 pmsrc/vendor/pocketmine/bedrock-protocol/src/ResourcePackClientResponsePacket(61): pocketmine\network\mcpe\handler\ResourcePacksPacketHandler->handleResourcePackClientResponse(object pocketmine\network\mcpe\protocol\ResourcePackClientResponsePacket#138120)
  #11 pmsrc/src/network/mcpe/NetworkSession(388): pocketmine\network\mcpe\protocol\ResourcePackClientResponsePacket->handle(object pocketmine\network\mcpe\handler\ResourcePacksPacketHandler#137998)
  #12 pmsrc/src/network/mcpe/NetworkSession(351): pocketmine\network\mcpe\NetworkSession->handleDataPacket(object pocketmine\network\mcpe\protocol\ResourcePackClientResponsePacket#138120, string[4] ....)
  #13 pmsrc/src/network/mcpe/raklib/RakLibInterface(182): pocketmine\network\mcpe\NetworkSession->handleEncoded(string[7] c.`a`..)
  #14 pmsrc/vendor/pocketmine/raklib-ipc/src/RakLibToUserThreadMessageReceiver(42): pocketmine\network\mcpe\raklib\RakLibInterface->onPacketReceive(int 0, string[16] ..XU#.!X..gi. t&E)
  #15 pmsrc/src/network/mcpe/raklib/RakLibInterface(112): raklib\server\ipc\RakLibToUserThreadMessageReceiver->handle(object pocketmine\network\mcpe\raklib\RakLibInterface#102691)
  #16 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(123): pocketmine\network\mcpe\raklib\RakLibInterface->pocketmine\network\mcpe\raklib\{closure}()
  #17 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
  #18 pmsrc/src/Server(1703): pocketmine\snooze\SleeperHandler->sleepUntil(float 1654589100.0617)
  #19 pmsrc/src/Server(1063): pocketmine\Server->tickProcessor()
  #20 pmsrc/src/PocketMine(304): pocketmine\Server->__construct(object BaseClassLoader#3, object pocketmine\utils\MainLogger#2, string[22] /home/edwinyoo44/pmmp/, string[30] /home /edwinyoo44/pmmp/plugins/)
  #21 pmsrc/src/PocketMine(327): pocketmine\server()
  #22 pmsrc(11): require(string[72] phar:///home/edwinyoo44/pmmp/PocketMine-MP_4.4.1.phar/src/PocketMine.php)
```

**Comment by dktapps** [comment]
I knew I should have used a singleton for this ...

**Comment by dktapps** [comment]
Fixed by 1584768c80f519b50b8dad96393c1a1ac6ce1382.
